### DISCUSSION TITLE
给 zsh 自定义命令添加参数自动补全 — 码志

### DISCUSSION THREAD

**Discussion by giscus** [mzlogin/blog-comments#270] [Announcements]
# 给 zsh 自定义命令添加参数自动补全 — 码志

有时我会自定义一些 zsh 命令，以便提升某些高频操作的效率。本文记录我给一个自定义命令添加参数自动补全的方法。

https://mazhuang.org/2022/02/12/zsh-completion/

**Comment by dom-bro** [comment]
666
